### PR TITLE
Add log axis for radar charts

### PR DESCRIPTION
**Pull request by zifix**
fixes https://github.com/apache/incubator-echarts/issues/11298

### CODE REVIEW

File: src/coord/radar/Radar.js [hunk 1]
```diff
@@ -40,7 +41,8 @@ function Radar(radarModel, ecModel, api) {
 
     this._indicatorAxes = zrUtil.map(radarModel.getIndicatorModels(), function (indicatorModel, idx) {
         var dim = 'indicator_' + idx;
-        var indicatorAxis = new IndicatorAxis(dim, new IntervalScale());
+        var indicatorAxis = new IndicatorAxis(dim,
+            (indicatorModel.axisType === 'log') ? new LogScale() : new IntervalScale());
```
Comment (pissang) [inline src/coord/radar/Radar.js]: Use `indicatorModel.get('type')` to access the `type` option in the `indicator` like this:
```js
indicator: [{ type: 'log' }]
```

**Comment by pissang** [comment]
Thanks.  

I've commented on the code. 

And please remove the distribution file. It will block us to merge your PR.

File: src/coord/radar/Radar.js (hunk 1 above)
Comment (zifix) [reply src/coord/radar/Radar.js]: I switched to get as requested.

**Comment by zifix** [comment]
> 
> 
> Thanks.
> 
> I've commented on the code.
> 
> And please remove the distribution file. It will block us to merge your PR.

I have removed both files in dist.

**Comment by pissang** [comment]
Hi @zifix

I meant not to commit these two distribution files. Not delete them.

**Comment by zifix** [comment]
> 
> 
> Hi @zifix
> 
> I meant not to commit these two distribution files. Not delete them.
I have readded the files.

**Comment by pissang** [comment]
Great! There is one last thing I think needs to be changed.

The `axisType` is better to be renamed to `type` to be same with other axis components.

**Comment by zifix** [comment]
I was thinking about that when I defined the attribute. I went with axisType because radar is not dedicated to axis attributes (look at e.g. shape). In the future you might like to have multiple radar types and the obvious attribute of choice is taken already.  
For perfect consistency, one might need to introduce a radarAxis attribute in some future api-breaking release.

**Comment by pissang** [comment]
Hi @zifix . Sorry for the late reply. I was on a vacation. I think what you explained is reasonable. Thanks a lot!

**Comment by Ovilia** [comment]
Moving to v4.5.0 because this PR has been merged before testing v4.5.0.

**Comment by johnKingwzw** [comment]
4.8好像又有了这个问题

**Pull request merged**